src/lib/containers/dataaccess/RejectDataAccessRequestModal.tsx

```diff
@@ -0,0 +1,422 @@
+import React, { useEffect, useState } from 'react'
+import {
+  Box,
+  Button,
+  Checkbox,
+  Collapse,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  FormControlLabel,
+  IconButton,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material'
+import IconSvg from '../IconSvg'
+import {
+  useGetFullTableQueryResults,
+  useUpdateDataAccessSubmissionState,
+} from '../../utils/hooks/SynapseAPI'
+import {
+  BUNDLE_MASK_QUERY_RESULTS,
+  REJECT_SUBMISSION_CANNED_RESPONSES_TABLE,
+} from '../../utils/SynapseConstants'
+import { Set } from 'immutable'
+import {
+  QueryResultBundle,
+  Row,
+  SubmissionState,
+} from '../../utils/synapseTypes'
+import { SynapseSpinner } from '../LoadingScreen'
+import { SynapseClientError } from '../../utils/SynapseClientError'
+import FullWidthAlert from '../FullWidthAlert'
+import { UseQueryResult } from 'react-query'
+import { ErrorBanner } from '../error/ErrorBanner'
+import { displayToast } from '../ToastMessage'
+
+const CATEGORY_COLUMN_NAME = 'category'
+const CATEGORY_SECTION_EMAIL_TEXT_COLUMN_NAME = 'category email prompt'
+const REJECTION_REASON_FORM_TEXT_COLUMN_NAME = 'rejection reason'
+const REJECTION_REASON_EMAIL_TEXT_COLUMN_NAME = 'email text'
+
+export type RejectDataAccessRequestModalProps = {
+  /* ID of the submission for which a rejection should be drafted/sent */
+  submissionId: string | number
+  /* SynID of the table which contains the email responses which should populate this modal. Defaults to syn50683097, which is in production Synapse */
+  tableId?: string
+  open: boolean
+  onClose: () => void
+}
+
+type RejectionCategoryProps = {
+  category: string
+  rows: Row[]
+  rejectionReasonFormTextIndex: number
+  selectedRowIds: Set<number>
+  setSelectedRowIds: React.Dispatch<React.SetStateAction<Set<number>>>
+}
+
+type SelectRejectionReasonsFormProps = {
+  tableQuery: UseQueryResult<QueryResultBundle, SynapseClientError>
+  selectedRowIds: Set<number>
+  setSelectedRowIds: React.Dispatch<React.SetStateAction<Set<number>>>
+}
+
+const DEFAULT_MESSAGE_PREPEND =
+  'Thank you for submitting your data access request.\n'
+const DEFAULT_MESSAGE_APPEND =
+  '\nIf you have questions, do not respond to this email address. Instead, reply to:\nact@sagebionetworks.org'
+
+/**
+ * Renders a rejection reason category and checkboxes for each reason in the category
+ */
+function RejectionCategory(props: RejectionCategoryProps) {
+  const {
+    category,
+    rows,
+    selectedRowIds,
+    setSelectedRowIds,
+    rejectionReasonFormTextIndex,
+  } = props
+  const [isExpanded, setIsExpanded] = React.useState(false)
+
+  return (
+    <>
+      <Typography
+        variant="body1"
+        onClick={() => setIsExpanded(!isExpanded)}
+        sx={{ fontWeight: 700, cursor: 'pointer', my: 1 }}
```

Comment:
```suggestion
        sx={{ fontWeight: 700, cursor: 'pointer', my: 1 }}
```

The `my` property is shorthand for **m**argin **y**-axis (top+bottom) using MUI's built-in spacing system: https://mui.com/system/spacing/